src/js/components/Jokes.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+export default class Jokes extends React.Component{
+    render(){
+        let jokes = this.props.jokes
```

Comment:
I used to use semicolons all the time, however, coming to use ES6, my understanding is not using it is safe. Please take a look at this repo https://github.com/rse/es6-features and let me know what you think.